.github/ISSUE_TEMPLATE/blank.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Open a blank issue.
+about: ''
+title: ''
+labels: needs-triage
+assignees: ''
+---
+
+<!-- 
+Please don't post screenshots of error messages; 
+we can't help with issues like that.
+Please instead copy and paste any error messages.
+-->
```

Comment:
What's the motivation for offering a blank template?

Reply:
This template is a replacement for the default blank issue template - I don't want to disable it completely. This one instead adds the triage label. It's a small improvement.